sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -631,13 +631,18 @@ public void populateDisplayData(DisplayData.Builder builder) {
         TableReference table = getTable();
 
         if (table != null) {
-          builder.add(DisplayData.item("table", toTableSpec(table)));
+          builder.add(DisplayData.item("table", toTableSpec(table))
+            .withLabel("Table"));
         }
 
         builder
-            .addIfNotNull(DisplayData.item("query", query))
-            .addIfNotNull(DisplayData.item("flattenResults", flattenResults))
-            .addIfNotDefault(DisplayData.item("validation", validate), true);
+            .addIfNotNull(DisplayData.item("query", query)
+              .withLabel("Query"))
+            .addIfNotNull(DisplayData.item("flattenResults", flattenResults)
+              .withLabel("Flatten Query Results"))
+            .addIfNotDefault(DisplayData.item("validation", validate)
+              .withLabel("Validate Paths"),
```

Comment:
Wouldn't this be validating the query, not the paths? Again, maybe "Validation Enabled" is clearer?